ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -214,29 +221,44 @@ gcp_file_system_terminate(WT_FILE_SYSTEM *file_system, WT_SESSION *session)
 }
 
 static int
-gcp_flush(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session, WT_FILE_SYSTEM *file_system,
-  const char *source, const char *object, const char *config)
+gcp_flush(WT_STORAGE_SOURCE * /*storage_source*/, WT_SESSION *session, WT_FILE_SYSTEM *file_system,
```

Comment:
Why are we using this approach for unused variables? C++17 provides a `[[maybe_unused]]` attribute. Could we use that here?

Reply:
Hi Tammy,
Jeremy suggested I use this approach instead of WT_UNUSED here since it ensures that the arguments can't be used in function.
I will change it to [[maybe_unused]]. 
Thanks!

Reply:
updated: 18c45a5272894954a5b9d94a241dd3c3f2636e32